docs/learn/tutorials/versioned/samza-rest-getting-started.md

```diff
@@ -48,7 +48,7 @@ Run the following commands:
 {% highlight bash %}
 cd samza-rest/build/distributions/
 mkdir -p deploy/samza-rest
-tar -xvf ./samza-rest-0.10.1-SNAPSHOT.tgz -C deploy/samza-rest
+tar -xvf ./samza-rest-0.11.0-SNAPSHOT.tgz -C deploy/samza-rest
```

Comment:
There is a merge conflict here. In current master this line is of this form: 
`tar -xvf ./samza-rest-0.11.1-SNAPSHOT.tgz -C deploy/samza-rest`. 
You can rebase with recent master to fix this.